modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/VirtualFlowTest.java

```diff
@@ -228,7 +234,8 @@ private void assertMatch(List<IndexedCell> control, AbstractList<IndexedCell> li
      * that the scroll bars and corner are not visible, and the clip view fills
      * the entire width/height of the VirtualFlow.
      */
-    @Test public void testGeneralLayout_NoCells() {
+    @Test
```

Comment:
This test still has few more unintentional @Test tag followed by newline changes.

Reply:
I fixed those, except for the ones that have a // comment in between @Test and the method. (Those were already there)